aea/agent.py

```diff
@@ -47,6 +46,10 @@ class Agent(AbstractAgent, WithLogger):
     }
     DEFAULT_RUNTIME: str = "threaded"
 
+    _runtime: BaseRuntime
```

Comment:
So the ABC is not aware of the concept of Runtime..

Reply:
it's just type hint for Agent class, cause runtime now defined in dedicated method, not init